CMakeLists.txt

```diff
@@ -72,6 +73,10 @@ else()
     message(STATUS "Disabling LSD support")
 endif()
 
+if(CURL_ENABLED)
+    list(APPEND VCPKG_MANIFEST_FEATURES "http")
```

Comment:
Do we want to keep the VCPKG feature called `http` or rename it to `curl` to be consistent with the CMake option?